### ISSUE TITLE
Settings: Control for Widgets 

### ISSUE THREAD

**Issue by l3enito** [pojome/elementor#1275]
We use Elementor (PRO) sine the beginning of the plugin and we build now our customers projects on Elementor!
And thats the feature request: Lots of great widgets 

-> BUT: once a website is finished,  the editors have to work with the pages and has to many choices, finding a widget is like  finding a "needle in the haystack". 

It is good when context menu is apearing after tickling a section or widget, but to exchange a widget in a given layout I need no more than 3 to 5 widgets for the editor. 

PLEASE add a list of widget with a wp role based grid to check or uncheck to "see" elementor widgets or the bunch of wp widgets. They should be always editable when they are used on pages and sections.

**Comment by arielk** [comment]
We call it "Advanced Role Manager". This feature is probably going to be a part of the Elementor PRO in the future.

I merged this issue with another issue #536 in the same subject so that we could deal with it more efficiently. 

Thanks for your feedback!

**Comment by nicholaszein** [comment]
Hello!

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.18.0, and improved with Elementor Pro 3.19.0! 🥳 

You can now use the Element Manager to decide what widgets should appear in the Editor and your website, and with Elementor Pro you can decide who can use those widgets. To learn more check the [documentation in the Help Center](https://elementor.com/help/element-manager/).

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
